services/venice-controller/src/main/java/com/linkedin/venice/controller/Admin.java

```diff
@@ -755,6 +756,12 @@ void updateRoutersClusterConfig(
    */
   boolean isParent();
 
+  /**
+   * Return the state of the parent controller in its fabric
+   * @return {@code ACTIVE} if the parent controller is actively serving requests in its fabric, else {@code PASSIVE}
```

Comment:
Instead of `{@code ACTIVE}`, can you change it to `{@link ACTIVE}`. Same in other places